docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1407,6 +1428,26 @@ The default is: `ETH`, `NET`, `WEB3`.
     The singular `--rpc-ws-api` and plural `--rpc-ws-apis` are available and are just two
     names for the same option.
 
+### rpc-ws-authentication-jwt-public-key-file
```

Comment:
Alphabetically this option needs to come after the other two `rpc-ws-authentication` options

Reply:
Done